6/changelog.yaml

```
@@ -94,3 +94,7 @@ releases:
         See `the removal process for details on how this works
         <https://github.com/ansible-collections/overview/blob/main/removal_from_ansible.rst#cancelling-removal-of-an-unmaintained-collection>`__
         (https://github.com/ansible-community/community-topics/issues/105).
+      - The servicenow.servicenow collection has been deprecated by its maintainers (https://github.com/ServiceNowITOM/servicenow-ansible/pull/69)
+        and will be removed from Ansible 7. It can still be installed manually, but it is suggested
+        to swich to `servicenow.itsm <https://galaxy.ansible.com/servicenow/itsm>`__ instead
```

```suggestion
        to swich to `servicenow.itsm <https://galaxy.ansible.com/servicenow/itsm>`__ instead
```
shouldn't there be one underscore? https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

---

Two underscores make sure you don't get warnings when the same link is used more than once.

---

Ref: https://github.com/ansible-community/ansible-build-data/pull/150

---

OK, thanks for clarifying!